code/modules/ghostroles/spawner/human/responseteams/ap_eridani.dm

```diff
@@ -0,0 +1,37 @@
+/datum/ghostspawner/human/ert/ap_eridani
+	name = "Eridani Asset Protection Specialist"
+	short_name = "eridaniassetprotection"
+	mob_name_prefix = "Spc. "
+	max_count = 2
+	desc = "A specialist suited for close asset protection and policing duties. Ensure your colleagues ledgers remain in the black."
+	welcome_message = "You are part of an Eridani Private Military Company Asset Protection Team, a highly trained group of security specialists and medical professionals \
+	contracted by the Stellar Corporate Conglomerate to protect its investments."
+	outfit = /datum/outfit/admin/ert/ap_eridani
+	possible_species = list(SPECIES_HUMAN, SPECIES_HUMAN_OFFWORLD)
+
+/datum/ghostspawner/human/ert/ap_eridani/commander
+	name = "Eridani Section Commander"
+	short_name = "eridanicommander"
+	mob_name_prefix = "Cmdr. "
+	max_count = 1
```

Comment:
That naming is imho a nogo, as it can easily be confused with the ERT / TCFL Commander (which is generally a ccia/admin role)
(Update the path aswell if you change it)

Reply:
Changed to Eridani Section Lead and updated all paths using it.